main.py

```diff
@@ -1,11 +1,46 @@
+import json
+
 import init_django_orm  # noqa: F401
+from db.models import Race, Skill, Guild, Player
+
+
+def get_race_instance(player: dict, player_race: str) -> Race:
+    race = Race.objects.get_or_create(
+        name=player_race,
+        defaults={"description": player["race"]["description"]}
+    )
+    return race[0]
```

Comment:
Perform unpacking instead of indexing
```suggestion
    race, _ = Race.objects.get_or_create(
        name=player_race,
        defaults={"description": player["race"]["description"]}
    )
    return race
```

Reply:
i agree it looks more nicely but is it better for computing in any way?

Reply:
ok, looks like to work faster

Reply:
Yes, it is better for computing since you avoid getting an object from the list by indexing and instead store the direct result in a variable.